api/src/test/java/org/pmiops/workbench/firecloud/FireCloudServiceImplTest.java

```diff
@@ -118,4 +126,36 @@ public void testIsUserMemberOfGroup_match() throws Exception {
         Lists.newArrayList(new ManagedGroupAccessResponse().groupName("group").role("member")));
     assertThat(service.isUserMemberOfGroup("group")).isTrue();
   }
+
+  @Test
+  public void testNihStatus() throws Exception {
+    NihStatus status = new NihStatus().linkedNihUsername("test").linkExpireTime(500L);
+    when(nihApi.nihStatus()).thenReturn(status);
+    assertThat(service.getNihStatus()).isNotNull();
+    assertThat(service.getNihStatus()).isEqualTo(status);
+  }
+
+  @Test
+  public void testNihStatusNotFound() throws Exception {
+    when(nihApi.nihStatus()).thenThrow(new ApiException(404, "Not Found"));
+    assertThat(service.getNihStatus()).isNull();
+  }
+
+  @Test(expected = ServerErrorException.class)
+  public void testNihStatusException() throws Exception {
+    when(nihApi.nihStatus()).thenThrow(new ApiException(500, "Internal Server Error"));
+    service.getNihStatus();
+  }
+
+  @Test
+  public void testNihCallback() throws Exception {
+    JWTWrapper wrapper = new JWTWrapper().jwt("random");
+    doNothing().when(nihApi).nihCallback(wrapper);
+    try {
+      service.postNihCallback(wrapper);
+    } catch (Exception e) {
+      fail();
+    }
```

Comment:
This test just confirms that service.postNihCallBack does not fail? is there anything that can be tested in this like if nihApi.nihCallback(wrapper) was called?

Reply:
I will flesh the tests out for this method in RW-1698, which will have code that actually uses it. For this PR (which doesn't use it), this is just a basic test and I was intentionally leaving out anything more complicated.